request_a_govuk_domain/request/utils.py

```diff
@@ -366,13 +366,15 @@ def phase_header_text(_request):
     :return:
     """
     phase = os.getenv("ENVIRONMENT")
-    if phase != "prod":
+    if phase not in ["prod", "stage"]:
         return {
+            "PHASE_COLOUR": "govuk-tag--pink",
```

Comment:
I think `PHASE_CLASS` is better as it's clearer what it is and could be something other than a colour (bold or italic)

Reply:
done and also updated the terms and the links